server/events/runtime/runtime.go

```diff
@@ -23,11 +24,16 @@ func MustConstraint(constraint string) version.Constraints {
 	return c
 }
 
+var invalidCharsInPlanFilenameRegex = regexp.MustCompile(`[^a-zA-Z0-9-_\.]`)
+
 // GetPlanFilename returns the filename (not the path) of the generated tf plan
 // given a workspace and maybe a project's config.
 func GetPlanFilename(workspace string, maybeCfg *valid.Project) string {
+	var unesacpedFilename string
```

Comment:
you've got a small typo here. It should be unes**ca**pedFilename not unesacpedFilename